src/java.net.http/share/classes/jdk/internal/net/http/common/HttpBodySubscriberWrapper.java

```diff
@@ -127,6 +134,15 @@ private void propagateError(Throwable t) {
      */
     protected void onCancel() { }
 
+    /**
+     * Called right after the userSubscriber::onSubscribe is called.
```

Comment:
Hello Daniel, I suspect this comment will need a change now, since the implementation in this PR now calls `onSubscribed` before the `userSubscriber::onSubscribe`.

Reply:
Ah! Good catch.